registry.json

```diff
@@ -2599,6 +2599,36 @@
       "documentation": "https://eth.wiki/json-rpc/API"
     }
   },
+  {
+    "id": "rootstock",
+    "name": "Rootstock",
+    "coinId": 137,
```

Comment:
please use 10000137 as id

Reply:
CoinId is available for RSK(Roostock) and not used by any other chain. 
Reference: 
https://github.com/satoshilabs/slips/blob/master/slip-0044.md

```

Coin type | Path                | Symbol | Coin
-----------------------------------------
137           | 0x80000089 | RBTC.    | RSK
```

 Feel free to share if you have different thought's to use `10000137 ` as id.